openapi.yaml

```diff
@@ -921,17 +921,25 @@ paths:
 
         The Well-Known URI is the entry point for clients and users, so make
         sure it is permanent and easy to use and remember. Clients MUST NOT
-        require the well-known path (`./well-known/openeo`) in the URL that is
-        specified by a user to connect to the back-end. A client MUST request
-        `https://example.com/.well-known/openeo` if a user tries to connect to
-        `https://example.com`. If the request to the well-known URI fails, the
-        client SHOULD try to request the capabilities at `/` from
-        `https://example.com`.
+        require the well-known path (`/well-known/openeo`) in the URL that is
```

Comment:
> `/well-known/openeo`

is dropping the leading `.` intentional here?

Reply:
```suggestion
        require the well-known path (`/.well-known/openeo`) in the URL that is
```

Reply:
No, thanks for catching this.